### DISCUSSION TITLE
Failure of Whitesource Renovate azure pipeline

### DISCUSSION THREAD

**Discussion by Sayali1638** [renovatebot/renovate#14577] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Renovate Me Plugin

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I am trying to run whitesource renovate using azure pipeline. I have followed [this](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) documentation.
My pipeline looks as follows:
![image](https://user-images.githubusercontent.com/66510021/157232102-03f025d9-8d32-4b1f-8eb9-dde1fe274241.png)

I have added the linux agent for the agent pool. When I try running the pipeline I face following error:
##[error]self signed certificate in certificate chain 

I have tried running it as mentioned but it is not working for me.

Please let me know how do I resolve the error. I am also not sure of where do I place the key provided by renovate.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by HonkingGoose** [comment]
I think this issue should be raised at https://github.com/jycouet/VSTSExtensions/issues.

Relevant part of our README:

> #### Renovate Me extension
> 
> Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
> From there you can create a pipeline with the `RenovateMe` task.
> [More details on how to configure the pipeline](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me).
> 
> **Note:** This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
